src/components/Transaction/Transaction.tsx

```diff
@@ -197,17 +215,31 @@ const Transaction = ({
                                 <CopyIcon className="w-3 h-3" />
                             </CopyTooltip>
                         )}
-                        <div className="flex justify-between mt-4 text-caption">
-                            <span className="capitalize text-primary-100">
-                                {t('transaction-detail')}
-                            </span>
+                        <div className="mt-4 text-caption capitalize">
+                            <span>{t('transaction-detail')}</span>
                         </div>
                     </>
                 )}
 
                 <div className="mt-1 text-tertiary" title={txId}>
                     {shortenHash(txId)}
                 </div>
+                <div className="pt-3 text-caption">
+                    <span className="capitalize">{t('transaction-memo')}</span>
+                    <div className="mt-1 text-tertiary">
+                        {decodedMemos?.map(
+                            (memo, index) =>
+                                memo && (
+                                    <span
+                                        className="break-words"
+                                        key={`${index}-${memo}`}
+                                    >
+                                        {`${memo.toString()} `}
```

Comment:
@ZilongRRR  Do we need to string here?

Reply:
remove in [6b04294](https://github.com/hashgreen/hoogii-wallet/pull/12/commits/6b04294cfd0d938902a0136911fff2add794bcf2)